src/eventViewer/components/TableRow.tsx

```diff
@@ -22,7 +23,7 @@ const TableRow: FC<Props> = ({row, style, fields}) => {
           } else if (Component) {
             content = <Component key={rowKey} row={row} />
           } else {
-            content = String(row[rowKey])
+            content = asLink(row[rowKey])
```

Comment:
Just making sure that it's intended for this content to be an array too?

Reply:
The issue wanted the data in this component handled the same as in the other case so I believe that is appropriate.  It works as I'd expect it to.